04portfolio-assignment/scripts/article.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+//create aticle object to hold all functions for the form
+var articles = [];
+
+//get all the data from blogsource
+function Article (sourceArt) {
+  this.title = sourceArt.title;
+  this.category = sourceArt.category;
+  this.author = sourceArt.author;
+  this.authorUrl = sourceArt.authorUrl;
+  this.publishedOn = sourceArt.publishedOn;
+  this.body = sourceArt.body;
+
+};
+
+//set the data to locations on index using handlebars and JQ
+Article.prototype.toHtml = function() {
+  var myCompile = Handlebars.compile($('#template').html());
+  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+  this.publishStatus = this.publishedOn ? `,   ${this.daysAgo} days ago` : '(draft)';
+  $('#articles').append(myCompile(this));
```

Comment:
You are appending the compiled template data in this function as well as where you call this function. Pick one place! :)   IMHO, you should be returning the complied data from this function, and append it below.